test/lib/usd/translators/UsdImportInstancesTest/ComplexScenario_optimized.usda

```diff
@@ -0,0 +1,145 @@
+#usda 1.0
+(
+    defaultPrim = "pCube1"
+    metersPerUnit = 0.01
+    upAxis = "Y"
+)
+
+def Xform "pCube1" (
+    kind = "component"
+    instanceable = true
+    prepend references = </InstanceSources/pCube1_instanceParent>
+)
+{
+    double3 xformOp:translate = (-4, 2, 0)
+    uniform token[] xformOpOrder = ["xformOp:translate"]
+}
+
+def Xform "pCube4" (
+    kind = "component"
+    instanceable = true
+    prepend references = </InstanceSources/pCube1_instanceParent>
+)
+{
+    double3 xformOp:translate = (4, 2, 0)
+    uniform token[] xformOpOrder = ["xformOp:translate"]
+}
+
+def Xform "pCube5" (
+    kind = "component"
+)
+{
+    double3 xformOp:translate = (0, 2, 0)
+    uniform token[] xformOpOrder = ["xformOp:translate"]
+
+    def "pCubeShape1" (
+        instanceable = true
+        prepend references = </InstanceSources/pCube1_pCubeShape1>
+    )
+    {
+    }
+
+    def "pCube2" (
+        instanceable = true
+        prepend references = </InstanceSources/pCube1_pCube2>
+    )
+    {
+    }
+
+    def "pCube3" (
+        instanceable = true
+        prepend references = </InstanceSources/pCube1_pCube3>
+    )
+    {
+    }
+
+    def Mesh "someUniqueChildOf_pCube5" {
+        double3 xformOp:translate = (0, -9, 0)
+        uniform token[] xformOpOrder = ["xformOp:translate"]
+
+        uniform bool doubleSided = 1
+        float3[] extent = [(-1, -1, 0), (1, 1, 0)]
+        int[] faceVertexCounts = [3]
+        int[] faceVertexIndices = [0, 1, 2]
+        point3f[] points = [(-1, -1, 0), (1, -1, 0), (0, 1, 0)]
+        color3f[] primvars:displayColor = [(0, 0, 0.2)]
+    }
+}
+
+over "InstanceSources"
```

Comment:
Cool, thanks @pmolodo. That all definitely makes sense to me, and I'm fine with leaving it as `InstanceSources`. I think I had that gut reaction just seeing the USD before I looked at the Python test script, and the intention is made a bit clearer in the docstrings there.

Reply:
I guess the other thing that isn't clear from this commit is that the name `InstanceSources` isn't an arbitrarily chosen name for the scope in these sample files, but the "standard" name that Maya creates when *exporting* instances:

https://github.com/Autodesk/maya-usd/blob/fab6650e8149f09fec1df28c6de0a5ae10c04d88/lib/mayaUsd/fileio/writeJobContext.cpp#L83

The more I think about it though - the more I think we should change this name to something more maya specific.  That way we can feel safer about treating it specially later... and if we are going to change it, better sooner rather than later.  (And it should be a very easy change to make...)

Reply:
Would that also affect the hardcoded "InstanceSources" in the AnimalLogic plugin?
plugin\al\lib\AL_USDMaya\AL\usdmaya\fileio\Export.cpp:296